packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -1265,3 +1292,57 @@ async function formatStakeDetails(db: DB, marketId: Address, stakeDetails: any[]
   }
   return formattedStakeDetails;
 }
+
+function getMarketsMeta(
+  marketsResults: any[],
+  filteredOutCount: number
+): MarketListMeta {
+  let categories = {};
+  for (let i = 0; i < marketsResults.length; i++) {
+    const marketsResult = marketsResults[i];
+    if (categories[marketsResult.category1]) {
+      categories[marketsResult.category1]['count']++;
+    } else {
+      categories[marketsResult.category1] = {
+        'count': 1,
+        'children': {},
+      };
+    }
+    if (categories[marketsResult.category1].children[marketsResult.category2]) {
+      categories[marketsResult.category1].children[marketsResult.category2]['count']++;
+    } else {
+      categories[marketsResult.category1].children[marketsResult.category2] = {
+        count: 1,
+        children: {},
+      };
+    }
+    if (categories[marketsResult.category1].children[marketsResult.category2].children[marketsResult.category3]) {
+      categories[marketsResult.category1].children[marketsResult.category2].children[marketsResult.category3]['count']++;
+    } else {
+      categories[marketsResult.category1].children[marketsResult.category2].children[marketsResult.category3] = {
+        count: 1,
```

Comment:
Does count for category3 ever increment?

Reply:
Yep, it gets incremented in the if statement above.